source/_posts/similarity.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
Link to   previous content - in content there should be link with previous content

Reply:
Done